eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -256,9 +256,9 @@ function ($identifier) {
     public function loadByRemoteId($remoteId)
     {
         return $this->getCacheValue(
-            $remoteId,
+            $this->escapeForCacheKey($remoteId),
             'ez-content-type-',
-            function ($remoteId) {
+            function () use ($remoteId) {
```

Comment:
Is this change necessary?

Reply:
without this change, we will use the variable after using the escapeForCacheKey function, and we need the original value to search in the database